3drepo/3drepo.io
# ISSUE #3342 - Realtime events - New/delete container

This fixes #3342

#### Description
- Add realtime event listener for when a new container is made
- Add realtime event listener for when a container is deleted

#### Test cases
- Open 3d repo in two accounts at the same time
- Navigate to the containers list (This also works in the Edit federation modal)
- In one account add a new container
- See if it appears in the other account
- Delete a container
- See if it disappears in the other account
